File: 황창식/java/class_practice/name_maker/src/name_maker/MyFileWrite.java
package name_maker;

import java.io.BufferedWriter;
import java.io.File;
import java.io.FileWriter;
import java.io.IOException;

public class MyFileWrite {
	public void fileWrite(String names[]) throws IOException {
		File file = new File("d:/names.txt");
		BufferedWriter bw = new BufferedWriter(new FileWriter(file));
		for (int i = 0; i < names.length; i++) {
			bw.write(names[i]);
			bw.write("/");
		}
		bw.close();
		
	}
}

File: 황창식/C_sharp/Class_practice_20210208/Class_practice_4/MyFileWrite.cs
using System;
using System.Collections.Generic;
using System.IO;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Class_practice_4
{
    class MyFileWrite
    {
        public void fileWrite(NameData[] arr)
        {
            using (StreamWriter outputFile = new StreamWriter($"C:/Users/KB/Desktop/SW_Lesson/C_sharp/Class_practice_20210208/names.txt"))
            {
                foreach (NameData name in arr)
                {
                    outputFile.WriteLine(name.FullName);
                    //outputFile.Write("/");
                }
            }
        }
    }
}
